Controller/SSO/RedirectToLogin.php

```diff
@@ -1,93 +1,87 @@
 <?php
 /**
- * @category    ClassyLlama
- * @package
- * @copyright   Copyright (c) 2019 Classy Llama Studios, LLC
+ * Copyright © Pixlee TurnTo, Inc. All rights reserved.
+ * See COPYING.txt for license details.
```

Comment:
Should this be Emplifi? If so, COPYING.txt would need updating.

Reply:
I've actually talked to legal about this a while ago, but they have not given me an answer. I've just been using this until told to use something different.